pos_default_empty_image/static/src/js/db.js

```diff
@@ -3,15 +3,6 @@ odoo.define('pos_default_empty_image.db', function (require) {
 
     var models = require('point_of_sale.models');
 
```

Comment:
@chienandalu I refactored that part with models.load_fields

Reply:
Ok, only the file should be `models.js`, isn't it?

Reply:
ok, I renamed it